src/main/java/org/snomed/snowstorm/rest/ExportController.java

```diff
@@ -38,12 +45,19 @@ public class ExportController {
 	@Autowired
 	private ModuleDependencyService moduleDependencyService;
 
+    @Autowired
+    private ExecutorService executorService;
+
 	@Operation(summary = "Create an export job.",
 			description = "Create a job to export an RF2 archive. " +
 					"The 'location' response header contain the URL, including the identifier, of the new resource.")
 	@PostMapping
-	public ResponseEntity<Void> createExportJob(@Valid @RequestBody ExportRequestView exportConfiguration) {
-		String id = exportService.createJob(exportConfiguration);
+	public ResponseEntity<Void> createExportJob(@Valid @RequestBody ExportRequestView exportRequestView) {
+		String id = exportService.createJob(exportRequestView);
```

Comment:
@dmcgihtsdo It is a good idea to use ExecutorService. Can you move these logic into the ExportService please?

Reply:
Thanks @CoderMChu, I have made this change. The method is now `exportRF2ArchiveAsync`.